StyleCop.Analyzers/StyleCop.Analyzers/Lightup/LightupHelpers.cs

```diff
@@ -280,5 +292,19 @@ internal static bool CanWrapNode(SyntaxNode node, Type underlyingType)
                     valueParameter);
             return expression.Compile();
         }
+
+        private static bool ValidatePropertyType(Type returnType, Type actualType)
+        {
+            if (returnType.GetTypeInfo().ImplementedInterfaces.Any(t => t.GetGenericTypeDefinition() == typeof(ISyntaxWrapper<>)))
```

Comment:
:bulb: Since we now have `WrapperHelper`, this method can be simplified:

```csharp
var requiredType = WrapperHelper.GetWrappedType(returnType) ?? returnType;
return requiredType == actualType;
```